js/ui/html_editor/ui.html_editor.js

```diff
@@ -545,23 +555,23 @@ const HtmlEditor = Editor.inherit({
     },
 
     /**
-    * @name dxHtmlEditorMethods.registerModules
-    * @publicName registerModules(modules)
+    * @name dxHtmlEditorMethods.register
+    * @publicName register(components)
     * @param1 modules:Object
     */
-    registerModules: function(modules) {
-        this._quillRegistrator.registerModules(modules);
+    register: function(components) {
+        this._quillRegistrator.registerModules(components);
 
         this.repaint();
     },
 
     /**
-    * @name dxHtmlEditorMethods.getModule
-    * @publicName getModule(modulePath)
+    * @name dxHtmlEditorMethods.get
+    * @publicName get(modulePath)
```

Comment:
We should use `component` here and in other cases, including the parameter passed to the `customize` function

Reply:
I've updated `get` function description.
N.B. `customize` works with the [modules](https://quilljs.com/docs/configuration/#modules) config only - it provides a way to customize of module config for our modules, or setup a custom module. So we can use `modules` as a parameter name

Reply:
Shouldn't we then rename `customize` to `customizeModules`? @TikhonovaTatyana

Reply:
I think `customizeModules` is a good name. 
Maybe call the parameter configs => customizeModules(configs)
@RomanTsukanov this parameter can look like the following:
```
{
    'history': {          
      'delay': 2500,
      'userOnly': true
    }, 
   'clipboard': { ... }
}
```

Reply:
Agreed.

Reply:
Done